astropy/coordinates/builtin_frames/ecliptic.py

```diff
@@ -10,11 +10,14 @@
 from .utils import EQUINOX_J2000
 
 __all__ = ['GeocentricTrueEcliptic', 'BarycentricTrueEcliptic',
-           'HeliocentricTrueEcliptic']
+           'HeliocentricTrueEcliptic', 'BaseEclipticFrame']
 
 
 class BaseEclipticFrame(BaseCoordinateFrame):
     """
+    A base class for frames that have names and conventions like that of
```

Comment:
Same comment as above for `BaseRADecFrame` - I use the Parameters block of the docstring to define the subclass docstrings.